recipes/mirror_maker.rb

```diff
@@ -0,0 +1,93 @@
+# coding: UTF-8
+# Cookbook Name:: cerner_kafka
+# Recipe:: mirror_maker
+
+# Create an array for storing errors we should verify before doing anything
+errors = Array.new
+
+# Verify node["kafka"]["mirror_maker"] is setup properly
+ruby_block 'assert mirror maker config is correct' do # ~FC014
+  block_name 'attribute_mirrormaker_assertions'
+  block do
+    # Sources
+    if node['kafka']['mirror_maker']['mirror_sources'].to_a.empty?
+      errors.push 'node[:kafka][:mirror_maker][:mirror_sources]  must not be empty'
+    else
+      srcs = node['kafka']['mirror_maker']['mirror_sources']
+      srcs.each do |c|
+        warn c
+        if !node['kafka']['mirror_maker'].has_key?(c)
+          errors.push 'node[:kafka][:mirror_maker][:' + c + '] entry in mirror_sources, but does not exist'
+        else 
+          if !node['kafka']['mirror_maker'][c].has_key?('zookeeper.connect')
+            errors.push 'node[:kafka][:mirror_maker][:' + c + '] entry must have zookeeper.connect attribute'
+          end
+          if !node['kafka']['mirror_maker'][c].has_key?('group.id')
+            errors.push 'node[:kafka][:mirror_maker][:' + c + '] entry must have group.id attribute'
+          end
+        end
+      end
+    end
+
+    # Target
+    if !node["kafka"]["mirror_maker"]["mirror_target.properties"].has_key?('metadata.broker.list')
```

Comment:
Nit: If we aren't making use of this I would say don't worry about it and leave it up to kafka to tell them what they did wrong